client/blocks/edit-gravatar/style.scss

```diff
@@ -1,19 +1,27 @@
 .edit-gravatar__image-container {
 	display: inline-block;
 	position: relative;
-	line-height: 0;
 
 	&.is-uploading::before {
 		content: '';
 		position: absolute;
 			top: 0;
-			right: 0;
-			bottom: 0;
+			right: 0; // needed for IE11
+			bottom: 0; // needed for IE11
```

Comment:
thanks for the nice comments!

of course, while at it, might there be more behind the need? why does IE11 require it? these kinds of explanations can help us in the future when we decide whether or not the workaround is still needed